utils/grafana-sap-hana-dashboard.json

```diff
@@ -2001,7 +2001,7 @@
       ],
       "targets": [
         {
-          "expr": "sort(sum(hanadb_schema_used_memory_mb{HOST=~\"$hana_node\"}) by (SCHEMA_NAME))",
+          "expr": "sort(sum(hanadb_schema_used_memory_mb{host=~\"$hana_node\"}) by (SCHEMA_NAME))",
```

Comment:
SCHEMA_NAME is still in uppercase. Is this correct? (i don't really know, just asking)